pyproject.toml

```diff
@@ -1,41 +1,56 @@
 [tool.poetry]
-name = "datastacks"
-version = "0.2.1"
-description = ""
-authors = ["Andy Durkan <andy.durkan@ensono.com>"]
+name = "pysparkle"
```

Comment:
Why named `pysparkle` rather than `datastacks`?

Reply:
to maintain consistency with what was deployed previously to databricks as a wheel. Also, pysparkle contents is what is used by Spark jobs, so I thought it is cleaner to have pysparkle wheel and import `from pysparkle`. rather than have datastacks wheel and import `from pysparkle`.

Reply:
Would it be much trouble to rename to datastacks and use the latter approach? To me it seems nicer to have the name be datastacks here, as it's the pyproject file for the whole project, rather than just the pysparkle part. But can live with it either way.

Reply:
Sorry @motylp I'm with Andy, datastacks says what it is considering it's used for the whole project. If you were an lay person to the project then pysparkle means nothing

Reply:
ok, it grew on me since yesterday, I can live with it - updated :)

Reply:
resolved